dotnet/src/IntegrationTests/Planning/StepwisePlanner/StepwisePlannerTests.cs

```diff
@@ -71,7 +71,7 @@ public void CanCreateStepwisePlan(bool useChatModel, string prompt, string expec
     [InlineData(true, "What is the tallest mountain on Earth? How tall is it divided by 2", "Everest")]
     [InlineData(false, "What is the weather in Seattle?", "Seattle", 1)]
     [InlineData(true, "What is the weather in Seattle?", "Seattle", 1)]
-    public async void CanExecuteStepwisePlan(bool useChatModel, string prompt, string partialExpectedAnswer, int expectedMinSteps = 1)
+    public async System.Threading.Tasks.Task CanExecuteStepwisePlan(bool useChatModel, string prompt, string partialExpectedAnswer, int expectedMinSteps = 1)
```

Comment:
Should there be an `Async` suffix?

Reply:
I made it so naming rules weren't enforced in test/sample code. But I'd gladly turn that on if folks are comfortable with that requirement.

Reply:
There are 82 violations of IDE1006 across test and sample code.